### ISSUE TITLE
Adding multiple authors requires an unintuitive pre-filled form deletion

### ISSUE THREAD

**Issue by hydrocheck** [hydroshare/hydroshare#3609]
When adding multiple authors, one must go back into the addition dialog for each author.  Further, the user must delete the previously-added user, even though the user is not actually being removed from the authors list.  This then clears up the search box to add another author.

[001.zip](https://github.com/hydroshare/hydroshare/files/3692695/001.zip)

**Comment by Maurier** [comment]
I think clearing the searchbox after adding an Author is a good improvement. However, having the modal window persist after this action is a kind of a trade-off. For resource owners who want to add a single Author it is ideal to have the modal window dismissed automatically. 

I'll wait for @dtarb's say in this matter to proceed.

**Comment by dtarb** [comment]
I agree with @Maurier.  Fix the problem of the search box not being clear when the modal is opened.  I do not think it is a problem for the modal to be dismissed and a user have to click + for each author they want to add.  However I do not feel strongly and would support improvements (but not at high priority).

Also - there should be consistency in the behavior between adding authors and adding contributors.  Right now there is not.  Groups can not be added as contributors (they should be).  The contributors modal does not remember the previous user (which is what we want for authors).  Similarity between the manage access and adding authors and contributors would also be advantageous for a consistent user experience.

**Comment by Maurier** [comment]
This is also related to https://github.com/hydroshare/hydroshare/issues/3700 and will be worked out as part of the same PR.
